### PR TITLE
Doc update core:set 

### PR DESCRIPTION
**Pull request by jpsiitonen**
Added example codes to methods which are defined in set. 
Currently there are no updates for Core:Set landing page ( already documented properly )

### CODE REVIEW

File: sdk/lib/core/set.dart [hunk 1]
```diff
@@ -140,6 +158,11 @@ abstract class Set<E> extends EfficientLengthIterable<E> {
   Iterator<E> get iterator;
 
   /// Whether [value] is in the set.
+  /// ```dart
+  /// final characters = {'A', 'B', 'C'};
+  /// final isB = characters.contains('B'); // true
```
Comment (tomgilder) [inline sdk/lib/core/set.dart]: nit

```suggestion
  /// final containsB = characters.contains('B'); // true
```

File: sdk/lib/core/set.dart [hunk 2]
```diff
@@ -140,6 +158,11 @@ abstract class Set<E> extends EfficientLengthIterable<E> {
...
+  /// final isD = characters.contains('D'); // false
```
Comment (tomgilder) [inline sdk/lib/core/set.dart]:
```suggestion
  /// final containsD = characters.contains('D'); // false
```

File: sdk/lib/core/set.dart [hunk 3]
```diff
@@ -168,12 +195,23 @@ abstract class Set<E> extends EfficientLengthIterable<E> {
...
+  /// ```
   void addAll(Iterable<E> elements);
 
   /// Removes [value] from the set.
   ///
   /// Returns `true` if [value] was in the set, and `false` if not.
   /// The method has no effect if [value] was not in the set.
+  /// ```dart
+  /// final characters = {'A', 'B', 'C'};
+  /// final removeB = characters.remove('B'); // true
```
Comment (tomgilder) [inline sdk/lib/core/set.dart]:
```suggestion
  /// final didRemoveB = characters.remove('B'); // true
```

File: sdk/lib/core/set.dart [hunk 4]
```diff
@@ -168,12 +195,23 @@ abstract class Set<E> extends EfficientLengthIterable<E> {
...
+  /// final removeD = characters.remove('D'); // false
```
Comment (tomgilder) [inline sdk/lib/core/set.dart]:
```suggestion
  /// final didRemoveD = characters.remove('D'); // false
```

File: sdk/lib/core/set.dart [hunk 5]
```diff
@@ -188,9 +226,19 @@ abstract class Set<E> extends EfficientLengthIterable<E> {
   /// rather than being based on an actual object instance,
   /// then there may not be a specific object instance representing the
   /// set element.
+  /// ```dart
+  /// final characters = {'A', 'B', 'C'};
+  /// final isB = characters.lookup('B');
```
Comment (tomgilder) [inline sdk/lib/core/set.dart]:
```suggestion
  /// final b = characters.lookup('B');
```

File: sdk/lib/core/set.dart [hunk 6]
```diff
@@ -188,9 +226,19 @@ abstract class Set<E> extends EfficientLengthIterable<E> {
...
+  /// print(isB); // B
```
Comment (tomgilder) [inline sdk/lib/core/set.dart]:
```suggestion
  /// print(b); // B
```

**Review by tomgilder — CHANGES_REQUESTED** [review]

**Review by alexlindroos — APPROVED** [review]

**Review by stepthomp — APPROVED** [review]

**Pull request closed without merging**